server/services/educationAndWorkPlanService.test.ts

```diff
@@ -87,48 +87,45 @@ describe('educationAndWorkPlanService', () => {
     it('should get Action Plan', async () => {
       // Given
       const prisonNumber = 'A1234BC'
-      const userToken = 'a-user-token'
       const actionPlanResponse = aValidActionPlanResponseWithOneGoal()
       educationAndWorkPlanClient.getActionPlan.mockResolvedValue(actionPlanResponse)
       prisonService.getAllPrisonNamesById.mockResolvedValue(new Map())
       const expectedActionPlan = aValidActionPlanWithOneGoal()
 
       // When
-      const actual = await educationAndWorkPlanService.getActionPlan(prisonNumber, userToken)
+      const actual = await educationAndWorkPlanService.getActionPlan(prisonNumber, systemToken)
```

Comment:
Not quite .... the service should be called with the username (ie. lines 96, 112, 125)

You assertions of how the client is called (lines 99, 115, 128) are correct 👍 , its just the above lines for the service calls that are wrong.

You could beef up these tests by adding an extra assertion, just after the ones you've changed:
```
expect(hmppsAuthClient.getSystemClientToken).toHaveBeenCalledWith(username)
```
That way you can be sure the auth client is being called with the correct expcted value ; ie. the username 👍

Reply:
You could add another assertion as well, just to be triple sure its all working as expected 😁 
You could add
```
expect(prisonService.getAllPrisonNamesById).toHaveBeenCalledWith(systemToken)
```

👍